meson.build

```diff
@@ -0,0 +1,48 @@
+# This file is adapted from https://github.com/scipy/scipy/blob/main/meson.build
+project(
+    'pandas',
+    'c', 'cpp', 'cython',
+    version: run_command(['python', 'generate_version.py', '--print'], check: true).stdout().strip(),
```

Comment:
Why not just always use python3 here?

Note that meson specifically has an easter egg reason to spell it with a 3, because, when searching for a command named literally `python3`, if meson can't find one available on $PATH it still knows one must exist -- because meson itself is written in it. So it can replace that with meson's own sys.executable as an emergency fallback. This means that scripts which only rely on the current source tree and the python standard library, are effectively guaranteed to work, always, no matter what.

It is rare for people to have multiple python 3.x available, not have a python3 command, *and* specifically want to use a different version of python from the one that meson is installed with. If it does happen, then most likely the "python" command either doesn't exist or doesn't do what you want either. The obvious example is BSD ports, which for some truly strange reason package many applications with one version of python and many others with a different version, then provide no "python" or "python3" command -- only "python3.X".